Dockerfile

```diff
@@ -20,7 +20,6 @@ RUN python3.8 -m pip install pip==${PIP_VERSION} && \
   pip install --no-cache -r requirements/base.txt && \
   pip install --no-cache -r requirements/test.txt && \
   pip install --no-cache -r requirements/dev.txt && \
-  pip install "unstructured.PaddleOCR" && \
```

Comment:
I guess we still use the paddle_ocr for tables?

Reply:
yep we do but maybe no need to keep it in dockerfile
1. we updated the forked library "unstructured.PaddleOCR" so that it doesn't use `paddlepaddle` as a dependency, so this paddle install here won't work since its missing a install for `paddlepaddle` ([updated way to install paddle](https://github.com/Unstructured-IO/unstructured/blob/main/scripts/install-paddleocr.sh)).
2. i don't think there are users using the inference dockerfile, and the install in docker will be moved to unstructured once we are ready to add it

Reply:
yes, not sure this Dockerfile should remain in the first place